patches/master_patch.patch

```diff
@@ -60,6 +60,19 @@ index c38d2a9fc72c177b5ee8e6bf05471c0758d5a757..d1cd1a1a29db9d7ae563b1b9724900f3
  _result = exec_script("version.py",
                        [
                          "-f",
+diff --git a/build/win/BUILD.gn b/build/win/BUILD.gn
+index 466e7ee84bf8cc45697def84f00af5c05f437f0f..ce083c782a4865ff0b955ab77b3cabe79b97383c 100644
+--- a/build/win/BUILD.gn
++++ b/build/win/BUILD.gn
+@@ -12,5 +12,8 @@ windows_manifest("default_exe_manifest") {
```

Comment:
actually I don't think we want this change at all. We just need to replace the deps in electron_app